### PR TITLE
rocketbank.ru

### PR DESCRIPTION
**Pull request by galeksandrp**

### CODE REVIEW

**Comment by J0WI** [comment]
There are a lot of more subdomains. Could you please check them?

**Comment by galeksandrp** [comment]
Added.

**Comment by J0WI** [comment]
I'm still missing some. Please also check these:
all.rocketbank.ru
arcada.rocketbank.ru
bless.rocketbank.ru
deposit.rocketbank.ru
estate.rocketbank.ru
holy.rocketbank.ru
kinohod.rocketbank.ru
pled.rocketbank.ru
profit.rocketbank.ru
quest.rocketbank.ru
samsungpay.rocketbank.ru
snow.rocketbank.ru

**Comment by galeksandrp** [comment]
Added.

File: src/chrome/content/rules/Rocketbank.ru.xml [hunk 1]
```diff
@@ -9,20 +12,30 @@ s3.rocketbank.ru ¹
 <ruleset name="rocketbank.ru">
 	<target host="rocketbank.ru" />
 	<target host="www.rocketbank.ru" />
+	<target host="all.rocketbank.ru" />
 	<target host="applepay.rocketbank.ru" />
-	<target host="artbattle.rocketbank.ru" />
```
Comment (J0WI) [inline src/chrome/content/rules/Rocketbank.ru.xml]: Why did you removed that one?

Comment (galeksandrp) [inline src/chrome/content/rules/Rocketbank.ru.xml]: Added back.

**Pull request merged**